docs/antora/modules/ROOT/pages/Common_Project_Layouts.adoc

```diff
@@ -336,6 +336,13 @@ This uploads them to `oss.sonatype.org` where you can log-in and stage/release
 them manually. You can also pass in the `--release true` flag to perform the
 staging/release automatically:
 
+[NOTE]
+--
+Sonatype credentials must have format `username:password` and must be passed as
+second argument or by named argument `--sonatypeCreds` or by specifying environment
+values `SONATYPE_USERNAME` and `SONATYPE_PASSWORD`.
+--
+
```

Comment:
In context of #2015 (security), we should probably suggest to prefer the environment variables, and only use the parameter as a fall back.

Reply:
Changed this note a little bit. I don't sure if we should mark this argument as deprecated or just note that environment variables are preferred?

Reply:
I think just the note is OK. Deprecating it would imply, that we are going to remove it someday, but that's not our intention.